packages/browser/src/browser/standalone-analytics.ts

```diff
@@ -1,30 +1,19 @@
-import { Analytics, InitOptions } from '../core/analytics'
 import { AnalyticsBrowser } from '.'
```

Comment:
Will changing this file name change the structure of the UMD build folder? If so, I would do this in a separate PR where we could evaluate the implications, as this is an actual entry point to the app and I would consider it to be a public API.